llvm/test/CodeGen/AMDGPU/lround.ll

```diff
@@ -14,6 +14,9 @@ declare i64 @llvm.lround.i64.f64(double)
 declare i64 @llvm.llround.i64.f32(float)
 declare half @llvm.round.f16(half)
 declare i32 @llvm.lround.i32.f16(half %arg)
+declare <2 x float> @llvm.round.v2f32.v2f32(<2 x float> %arg)
+declare <2 x i32> @llvm.lround.v2i32.v2f32(<2 x float> %arg)
+declare <2 x i64> @llvm.lround.v2i64.v2f32(<2 x float> %arg)
```

Comment:
The `declare` s are a waste of your time:
https://discourse.llvm.org/t/recent-improvements-to-the-ir-parser/77366

Reply:
I will remove the lines